src/overlays/actors/ovl_Obj_Hsblock/z_obj_hsblock.c

```diff
@@ -14,8 +14,12 @@ void ObjHsblock_Init(Actor* thisx, GlobalContext* globalCtx);
 void ObjHsblock_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void ObjHsblock_Update(Actor* thisx, GlobalContext* globalCtx);
 void ObjHsblock_Draw(Actor* thisx, GlobalContext* globalCtx);
+void func_80B93D90(ObjHsblock* this);
```

Comment:
line break after main funcs
I also like the group the action functions together and then put the other stuff below it but thats preference

Reply:
I wasn't sure where we landed on this. @Roman971 thoughts?

Reply:
I agree with line breaks between "main" functions and the rest

Reply:
so not grouping action functions in one group? i think it makes sense to do this rather then having it mixed with other stuff inbetween

Reply:
I do too but I recall Roman saying they weren't significant differences or something in another PR before. Maybe I'm mis-remembering the specifics though.

Reply:
that was on the topic of line breaking other update/draw functions that also take a thisx

Reply:
grouped 'em

Reply:
I think that would be a documentation step at least